pages/index.vue

```diff
@@ -16,6 +16,16 @@
         </nuxt-link>
       </template>
     </DHero>
+    <template>
```

Comment:
remove the `template`, maybe the `div` too (if keeping the div, change the id to something more specific)